client/src/components/campaign/round-new/round-new.html

```diff
@@ -0,0 +1,165 @@
+                  </md-select>
+                </md-input-container>
+              </div>
+            </div>
+
+            <h4 ng-if="$ctrl.index === 1">Source Category</h4>
+            <p class="muted" ng-if="$ctrl.index === 1">Category on Wikimedia Commons that gathers all contest images. Example of such category may be eg. "Images from Wiki Loves Monuments 2017 in Ghana".</p>
+            <md-autocomplete flex
+              ng-if="$ctrl.index === 1" required
+              md-input-name="autocompleteField"
+              md-input-minlength="2"
+              md-input-maxlength="500"
+              md-autoselect="true"
+              md-selected-item="$ctrl.category"
+              md-search-text="$ctrl.categorySearchText"
+              md-items="item in $ctrl.searchCategory($ctrl.categorySearchText)"
+              md-item-text="item"
+              md-no-cache="true"
+              md-floating-label="Enter category"
+              md-delay="100">
+              <md-item-template>
+                <span md-highlight-text="$ctrl.searchText">{{ item }}</span>
+              </md-item-template>
+            </md-autocomplete>
+
+            <h4>Directions</h4>
+            <md-input-container class="no-margin">
+              <textarea aria-label="Round directions"
+                ng-model="$ctrl.round.directions"
+                rows="2" ></textarea>
+            </md-input-container>
+          </div>
+          <div flex></div>
+          <div class="round__settings" flex="30">
+            <h4>Round File Settings</h4>
+            <p layout="column" layout-align="start stretch">
+              <span layout="row" layout-align="space-between start"
+                  ng-repeat="(name, value) in $ctrl.round.config">
+                <span ng-if="name !== 'min_resolution'">{{ ::name | translate }}</span>
+                <span ng-if="value === false || value === true">
+                  <md-checkbox
+                    ng-model="$ctrl.round.config[name]"
+                    aria-label="{{ ::name }}">
+                </span>
+                <md-input-container ng-if="name === 'min_resolution'">
+                  <label>Minimal resolution</label>
+                  <input type="number" ng-model="$ctrl.round.config.min_resolution">
+                </md-input-container>
+              </span>
+            </p>
+          </div>
+        </div>
+        
+        <div class="round__jurors"
+            layout="row" layout-align="center stretch">
+          <div flex>
+            <h4>Quroum</h4>
+            <p class="muted">The number of jurors that must vote on each image</p>
+            <md-input-container class="no-margin-bottom">
+              <label>Quorum</label>
+              <input type="number"
+                  ng-model="$ctrl.round.quorum"
+                  ng-required="true">
+            </md-input-container>
+            <h4>Jurors</h4>
+            <mont-user-list
+              ng-model="$ctrl.round.jurors">
```

Comment:
I see you have to click the juror name from the list, and you can't just type in their name. This isn't super intuitive at first: it can be a bit confusion if you type in a name and then move to another field, but can't create the round. Is there a way to mark this field as highlighted/invalid if you input text but don't select a juror?